messages/de/app.php

```diff
@@ -41,11 +41,11 @@
     'Discord servers partnered with <span class="brand">DISBOARD</span>' => '<span class="brand">DISBOARD</span> Partner',
     'Discord Server List' => 'Discord Serverliste',
     'Discord Server Reviews' => 'Discord Server Rezensionen',
-    'Discord Servers Bumped Recently' => 'Kürzlich gebumpte Discord Server',
-    'Discord servers categorized as <strong>{category}</strong>' => 'Discord Server aus der Kategorie <strong>{category}</strong>',
-    'Discord servers tagged with {tag}' => 'Liste der Discord Server mit dem Stichwort {tag}',
-    '<span class="discord-logo">Discord</span> Servers' => '<span class="discord-logo">Discord</span> Server',
-    'Discord servers tagged with <strong>{tag}</strong>' => 'Discord Server mit dem Stichwort <strong>{tag}</strong>',
+    'Discord Servers Bumped Recently' => 'Kürzlich gebumpte Discord Servers',
```

Comment:
Both [Wiktionary](https://de.wiktionary.org/wiki/Server) and the [Duden](https://www.duden.de/deklination/substantive/Server) define the plural of “der Server” as “die Server”, making this (and any adjustments below this where “Server” is being changed to “Servers”) invalid.

Reply:
Ich hab aufs Sprachgefühl gehört sry